test/main.js

```diff
@@ -133,6 +170,37 @@ const createMocha = (driver, testCase) => {
   // By default `require` caches files, making it impossible to require the same file multiple times.
   // Since we want to execute the same tests against many browsers we need to prevent this behaviour by
   // clearing the require cache.
+  mocha.suite.beforeAll('Do something before everything', function () {
```

Comment:
Might be worth adding a more meaningful message here
```suggestion
  mocha.suite.beforeAll('Setting up browser before tests execution, function () {
```